src/templates/post.html

```diff
@@ -19,9 +19,7 @@ <h1>$title$</h1>
   <div class="layout--constrained">
     <ul role="list">
       <li role="listitem">
-        <span aria-hidden="true">♥</span>
-        <a href="https://github.com/sponsors/rpearce/">Sponsor my work</a>
-        <span aria-hidden="true">♥</span>
+        <span aria-hidden="true">♥</span> <a href="https://github.com/sponsors/rpearce/">Sponsor my work</a> <span aria-hidden="true">♥</span>
```

Comment:
I love that putting these on a single line with a space in-between preserves that whitespace! Exactly what I wanted.